easybuild/easyconfigs/d/deap/deap-0.9.2-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'deap'
+version = '0.9.2'
+
+homepage = 'http://deap.readthedocs.org/en/master/'
+description = """DEAP is a novel evolutionary computation framework for rapid prototyping and testing of ideas.
+ It seeks to make algorithms explicit and data structures transparent."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [PYPI_SOURCE]
```

Comment:
It's been there for quite a while already (Dec'13, see https://github.com/hpcugent/easybuild-framework/pull/793)